File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/arcade_drive/RedArcadeDrive.java
package org.firstinspires.ftc.teamcode.teleops.arcade_drive;

import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.firstinspires.ftc.teamcode.Side;

@TeleOp(name = "Red Arcade Drive", group = "Competition")
public class RedArcadeDrive extends ArcadeDriveBase {
    @Override
    protected Side getSide() {
        return Side.RED;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/arcade_drive/RedArcadeDrive.kt
package org.firstinspires.ftc.teamcode.teleops.arcade_drive

import com.qualcomm.robotcore.eventloop.opmode.Disabled
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import org.firstinspires.ftc.teamcode.utils.Side

@TeleOp(name = "Red Arcade Drive")
@Disabled
class RedArcadeDrive : ArcadeDriveBase() {
    override val side: Side = Side.RED
}